packages/datepicker-react/src/DatePicker.test.tsx

```diff
@@ -479,21 +540,18 @@ describe("after user types string", () => {
 });
 
 describe("a11y", () => {
-    it("default datepicker should be a11y compliant", async () => {
+    it("datepicker should be a11y compliant", async () => {
         const { container } = render(<DatePicker initialShow />);
-        const results = await axe(container);
+        const collapsed = await axe(container);
 
-        expect(results).toHaveNoViolations();
-    });
+        expect(collapsed).toHaveNoViolations();
 
-    it("extended datepicker should be a11y compliant", async () => {
-        const { container } = render(<DatePicker extended initialShow />);
```

Comment:
Noen grunn til at du ikke lenger vil teste denne varianten? Nå ser det i hvert fall ut som den tester den vanlige to ganger. Husk at `extended` her ikke betyr "åpen", men varianten med nedtrekksliste for måned, og inputfelt for årstall :)

Reply:
Ah, her sakset jeg bort en linje for mye. Mente å re-rendre komponenten med extended-prop. Grunnen til at jeg ville slå dem sammen var at jeg på et tidspunkt hadde en tredje test som timet ut. Da feilet en av disse med at en `axe`-sjekk allerede var underveis, av en eller annen grunn. Tenkte det var en race condition som lå på lur, så ville slå de sammen for å være garantert bare én sjekk av gangen.

Kan kanskje bare reverte denne endringen av a11y-tester.